toolkit/docs/quick_start/quickstart.md

```diff
@@ -0,0 +1,128 @@
+# Quick Start Guide
+  - [Install Prerequisites](#install-prerequisites)
+  - [Clone CBL-Mariner](#Clone-CBL-Mariner)
+  - [Build and Boot an Image](#build-and-boot-an-image)
+    - [VHDX and VHD Images](#vhdx-and-vhd-images)
+    - [ISO Image](#iso-image)
+  
+## **Install Prerequisites**
+Install prerequisites [here](../building/prerequisites.md).
+
+ 
+## **Clone CBL-Mariner**
+From a bash terminal window, clone the CBL-Mariner Repository and check-out a stable build.
+
+```bash
+# Clone the CBL-Mariner repo
+git clone git@github.com:microsoft/CBL-Mariner.git
+cd CBL-Mariner
+
+# Sync to the latest stable build 
+git checkout 1.0-stable
+
+```
+
+## **Build and Boot an Image**
+
+### **VHDX and VHD images**
+
+The following builds a bootable, VHDX or VHD CBL-Mariner image from precompiled RPMs in the CBL-Mariner package repository at https://packages.microsoft.com/cbl-mariner/1.0/prod/.
+
+No user account is provisioned by default.  To sign-in to these images, the sample meta-user-data.iso image must also be built and installed in your VM's CD drive.  The cloud-init service will detect the iso and provision a user account and password.  
+
+**Build a VHD or VHDX Image**
+```bash
+# Switch to the toolkit folder 
+cd toolkit
+
+# Build VHDX Image
+# Image is placed in ../out/images/core-efi
+sudo make image REBUILD_TOOLS=y REBUILD_PACKAGES=n CONFIG_FILE=./imageconfigs/core-efi.json
+
+# Build VHD Image
+# Image is placed in ../out/images/core-legacy
+sudo make image REBUILD_TOOLS=y REBUILD_PACKAGES=n CONFIG_FILE=./imageconfigs/core-legacy.json
+```
+
+**Build the cloud-init configuration image**
+```Bash
+# Build the cloud-init configuration image 
+# The output image is ../out/images/meta-user-data.iso
+sudo make meta-user-data
+
+```
+
+**Copy VHD(X) and ISO Images to Your VM Host Machine**
+
+**Create VHD(X) Virtual Machine with Hyper-V**
+
+1. From Hyper-V Select _Action->New->Virtual Machine_.
+1. Provide a name for your VM and press _Next >_.
+1. For VHD select _Generation 1_, For VHDX select _Generation 2_, then press _Next >_.
+1. Change Memory size if desired, then press _Next >_.
+1. Select a virtual switch, then press _Next >_.
+1. Select _Use an existing virtual hard disk_, then browse and select your VHD(X) file.
+1. Press _Finish_.
+
+**Mount the Meta-User-Data.Iso Image**
+
+1. Right click your virtual machine from Hyper-V Manager
+1. Select _Settings..._.
+1. For Gen2/VHDX Images only, select Security and disable _Enable Secure Boot_.
+1. For Gen2/VHDX Images only, select the SCSI Controller from the Hardware panel, choose DVD Drive and press Add.
```

Comment:
Suggestion: for these two steps we could **bold** out the fact that it's only required for VHDXs, so it's easier to see and immediately skip. Example:
1. **[Gen2/VHDX only]** Select _Security_ and disable _Enable Secure Boot_.

Reply:
Another thought: maybe move the two steps to a separate optional section for VHDXa, since this is not part of mounting `meta-user-data.iso` anyway and can be easily missed if someone already mounted it and is trying to figure out what's still wrong.

---
In reply to: [546941391](https://github.com/microsoft/CBL-Mariner/pull/476#discussion_r546941391) [](ancestors = 546941391)

Reply:
I like both ideas.  I added the bolding and shifted the "Image Creation" details to the section above. I did not make a whole new section, but I did append it so it's inline with the creation details